cedar-policy-symcc/src/err.rs

```diff
@@ -15,35 +17,41 @@
  */
 use thiserror::Error;
 
-#[derive(Debug, Error)]
+pub use crate::{
+    solver::SolverError,
+    symcc::{BitVecError, CompileError, ConcretizeError, DecodeError, IPError},
+};
+
+/// Top-level errors from the whole `cedar-policy-symcc` crate.
+#[derive(Debug, Diagnostic, Error)]
+#[diagnostic(
+    help("if you are not using *_unchecked functions, this is likely due to a bug in the symbolic compiler or cvc5")
+)]
```

Comment:
Is this true for all of the variants here -- I'm particularly looking at `SolverError` which can be thrown in different ways depending on the particular `Solver` implementation?

Reply:
Ah I see. Maybe I'll remove this help message for now